scripts/india_plfs/daily_wage_data/README.md

```diff
@@ -0,0 +1,82 @@
+# Periodic Labour Force Survey (PLFS) -  Average daily wage earnings
+
+
+## About the Dataset
+Average wage earnings (Rs. 0.00) per day from casual labour work other than public works in CWS for each State/ UT.
```

Comment:
nit: expand CWS or link to what it means?

Reply:
It's already there in the **Notes from the data page :** section.

Reply:
**Notes from the data page :** 
1. The Current Weekly Status (CWS) approach to measuring uses seven days preceding the date of survey as the reference period.